libmamba/include/mamba/core/solver.hpp

```diff
@@ -86,6 +89,8 @@ namespace mamba
         bool force_reinstall = false;
 
     private:
+        static void delete_libsolve_solver(Solver* solver);
```

Comment:
Looks like this should be a free function in the cpp? The unique-ptr only need `void(Solver*)` type.